core/src/main/scala/org/dbpedia/extraction/mappings/InfoboxMappingsExtractor.scala

```diff
@@ -0,0 +1,161 @@
+        case _ => ""
+      }
+    } else if (typeOfStr == "#invoke") {
+     val list_words = str.split("""\|""")
+      if (!(list_words(0) == "Wikidata" || list_words(0) == "PropertyLink"))
+        return answer
+      val properties = list_words.filter(s => (s.charAt(0) == 'p' || s.charAt(0) == 'P') && isNumber(s.substring(1)))
+      for ( p <- properties){
+        answer = answer + "/" + p
+      }
+
+      if (answer.size > 0)
+       answer = answer.substring(1)
+
+    }
+    answer
+  }
+  def getPropertyTuples(page : PageNode) : List[(String,String, String)] = {
+    val parserFunctions = ExtractorUtils.collectParserFunctionsFromNode(page)
+
+    val propertyParserFunctions = parserFunctions.filter(p => (p.title.equalsIgnoreCase("#property") &&  //To filter out parser functions with title #property
+      p.children.nonEmpty && // Ignore those that have no children
+      !p.children.head.toString.contains("from") && // Ignore those that have "from" in them for eg {#property:P1308|from=Q824910}
+      p.parent.isInstanceOf[PropertyNode])) // Parent needs to be a PropertyNode to get the key
+
+     (propertyParserFunctions ).map( p =>
+      new Tuple3(p.parent.asInstanceOf[PropertyNode].parent.asInstanceOf[TemplateNode].title.decoded, p.parent.asInstanceOf[PropertyNode].key, extract_property(p.toWikiText, "#property")))
+
+  }
+
+   def getInvokeTuples(page : PageNode) : List[(String, String, String)] = {
+    val parserFunctions = ExtractorUtils.collectParserFunctionsFromNode(page)
+    var invokeFunc = parserFunctions.filter(p => ( p.title.equalsIgnoreCase("#invoke")))
+     invokeFunc = invokeFunc.filter(p => extract_property(p.children.head.toWikiText, "#invoke") != "")
+    (invokeFunc ).map( p => new Tuple3(p.parent.asInstanceOf[PropertyNode].parent.asInstanceOf[TemplateNode].title.decoded, p.parent.asInstanceOf[PropertyNode].key, extract_property(p.children.head.toWikiText, "#invoke")))
+
+  }
+
+  def getDirectTemplateWikidataMappings(propertyNode : PropertyNode, lang: Language) : Boolean = {
+
+    for( x <- directTemplateMapsToWikidata.getOrElse(lang.wikiCode, Map())){
+      if (reduceChildrenToString(propertyNode).contains(x._1))
+        return true
+    }
+
+    return false
+
+  }
+
+  def getP856Tuples(page : PageNode, lang : Language) : List[(String, String, String)] = {
+    val templateNodes = ExtractorUtils.collectTemplatesFromNodeTransitive(page)
+    val infoboxes = templateNodes.filter(p => p.title.toString().contains(infoboxNameMap.get(lang.wikiCode).getOrElse("Infobox")))
+
+    var website_rows = scala.collection.mutable.ListBuffer[PropertyNode]()
+    infoboxes.foreach(x => {
+      website_rows = website_rows ++ x.children.filter(p => getDirectTemplateWikidataMappings(p, lang) )
+    })
+    var answer = scala.collection.mutable.ListBuffer[(String, String, String)]()
+    for ( x <- website_rows){
+     answer += new Tuple3(x.parent.asInstanceOf[TemplateNode].title.decoded, x.key.toString, "P856")
```

Comment:
you also need to change this and get the value of the map (https://github.com/dbpedia/extraction-framework/pull/459/files#diff-344bbce058d6e918bd4e645072b51268R15)